### ISSUE TITLE
Update mesher for new file format

### ISSUE THREAD

**Issue by aevitas** [BosslandGmbH/BuddyWing#17]
Couple patches ago the file format changed which broke the mesher. We'll need to update the mesher for the new file format to be able to service meshing requests again.

**Comment by aevitas** [comment]
@pushedx PTAL

**Comment by pushedx** [comment]
Not sure if you saw these skype messages but I'll copy them for reference.

[2/25/2016 3:13:32 PM] Drew Benton: thanks got the swtor file, should be able to see what changed with that soon
[2/25/2016 3:15:45 PM] Drew Benton: don't think I can see that tl repo, but I had some code I was going to mess with later to give that a try, you basically do what HSB is doing with unity, load up the api of core calls, then callinjected on them. I had a demo eve project that did that a while back and it worked fine, so it should work for WoT too, however don't really have time to mess with it for now
[2/25/2016 3:23:13 PM] Drew Benton: what you sent me, was that part of the last working mesh files?
[2/25/2016 3:28:09 PM] Drew Benton: this one isn't working either and it's roughly the same as the current files so might need one from even earlier, but can't find any online
[2/25/2016 6:26:14 PM] Drew Benton: I'm working on reversing the new file, and making progress but I really need one of the old files to see about updating what broke. Basically when this stuff did break, it was because they switched from a text format to a binary format, so the text format processing stuff in the tools doesn't work correctly. https://gist.github.com/pushedx/25c51fd635a6d4e4832e
[2/25/2016 7:47:56 PM] Drew Benton: updated the paste, can't seem to find AreaDisplayName and AreaGUID anymore, so i'll see if it's really needed or not then start checking out the other files that broke
[2/26/2016 4:22:09 PM] Drew Benton: alright so made some progress on the rooms file, but it's using a more complex format I'll have to work out somehow I'll keep at in on and off going into eb update

At the last message I worked out the area display name and guid stuff. The extra bytes unprocessed weren't used for anything and they represent minor unuseful stuff so won't spend the time on that.

I won't be able to do anything more for this for about a month (best case), as that's about how long EB is going to take to get back up and running with their new client.

I tried searching around for old SWTOR installs and files to get old data, but no luck. That would greatly help speed up things, as otherwise I'll just have to do a ton of debugging and try and work out the new rooms format.
